etc/picongpu/summit-ornl/gpu_picongpu.profile.example

```diff
@@ -18,22 +18,22 @@ export proj=<yourProject>
 #export EDITOR="nano"
 
 # basic environment ###########################################################
-module load gcc/6.4.0
+module load gcc/8.1.1
```

Comment:
Brave of you, GCC 8.0-8.3 is very bug riddled (8.4+ is fine again) :-)

Reply:
Thanks, I was switching to the newest compiler which is supported by CUDA 10.2 and provides all modules we need. I will have a look if 8.4+ is available with all required modules.